### PR TITLE
[Validator] Add the missing translations for the Welsh (cy) locale

### PR DESCRIPTION
**Pull request by houdmont**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #30154
| License       | MIT

Addresses the missing translations listed in #30154

### CODE REVIEW

**Comment by houdmont** [comment]
I'm not sure it should have the "Deprecation" label.

File: src/Symfony/Component/Validator/Resources/translations/validators.cy.xlf [hunk 1]
```diff
@@ -222,6 +222,114 @@
...
+                <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
+                <target>Mae'r ddelwedd mewn fformat portread ({{ width }}x{{ height }}px). Ni chaniateir delweddau mewn fformat portread.</target>
+            </trans-unit>
+            <trans-unit id="78">
+                <source>An empty file is not allowed.</source>
+                <target>Ni chaniateir ffeil wag.</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>>Ni fu modd datrys y gwesteiwr.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.cy.xlf]: Here there is a minor issue: `<target>>` -> `<target>`

**Review by javiereguiluz — APPROVED** [review]
Great contribution! Thanks a lot Tom.

New messages were checked with Google Translate and no error was found.

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @houdmont.

**Pull request merged**